pkg/sessions/dynamodb/client.go

```diff
@@ -0,0 +1,74 @@
+package dynamodb
+
+import (
+	"context"
+	b64 "encoding/base64"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	dynamo "github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"time"
+)
+
+type Client struct {
+	*dynamo.DynamoDB
```

Comment:
By embedding this, the `Client` will inherit all of the methods from the `DynamoDB` which I don't think we want to be doing in this case, you can make it a member instead
```suggestion
	dynamoService *dynamo.DynamoDB
```

I think we can add the table name as a member of this struct too?